engine/storage_utils.go

```diff
@@ -84,22 +81,19 @@ func NewStorDBConn(dbType, host, port, name, user, pass, marshaler string,
 	return
 }
 
-func buildURL(scheme, host, port, db, user, pass string) (*url.URL, error) {
-	u, err := url.Parse("//" + host)
-	if err != nil {
-		return nil, err
+func composeURI(scheme, host, port, db, user, pass string) string {
```

Comment:
If this is only used for mongo, please rewrite the function to mongoURI().
Also please do not forget to always add comments to the functions.

Thanks!